src/data-types-lib/fieldTypes/decimal/stringUiType.js

```diff
@@ -11,23 +11,25 @@ export default {
    * ████  FIELD_TYPE_DECIMAL  ►  UI_TYPE_STRING  ████
    * █████████████████████████████████████████████████
    */
-  format: value => String(value),
+  format: (value, { i18n } = {}) => value === EMPTY_FIELD_VALUE ?
```

Comment:
Default value does not make any sense because
1. **i18n** is queried with **formatDecimalNumer()**.
2. **i18n** is required according to **src/crudeditor-lib/index.js***

Change arguments from `(value, { i18n } = {})` to `({ value, i18n })`.